sendgrid/helpers/mail/email.py

```diff
@@ -35,6 +49,14 @@ def name(self):
 
     @name.setter
     def name(self, value):
+        if not (value is None or isinstance(value, str)):
+            raise TypeError('name must be of type string.')
+
+        # Escape common CSV delimiters as workaround for
+        # https://github.com/sendgrid/sendgrid-python/issues/578
+        if value is not None and (',' in value or ';' in value):
+            value = html_entity_decode(value)
```

Comment:
I think you actually want [`email.utils.quote`](https://docs.python.org/3/library/email.util.html#email.utils.quote) here. (Email display names don't really have anything to do with HTML entities.)